.github/workflows/ci.yaml

```diff
@@ -0,0 +1,27 @@
+name: CI
+
+on:
+  push:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: 1.21
```

Comment:
Would `go-version-file` to `go.mod` be better? therefore tracking of go versions is in one single place and maintainers don't need to ensure that they're updating said action each time?

Reply:
```suggestion
          go-version-file: "go.mod"
```